webrender/src/debug_render.rs

```diff
@@ -59,63 +62,94 @@ const DESC_COLOR: VertexDescriptor = VertexDescriptor {
         },
     ],
     instance_attributes: &[],
-};
+};*/
 
 #[repr(C)]
+#[derive(Copy, Clone)]
 pub struct DebugFontVertex {
     pub x: f32,
     pub y: f32,
-    pub color: ColorU,
+    z: f32,
+    pub color: ColorF,
     pub u: f32,
     pub v: f32,
+    s: f32,
```

Comment:
why do we need extra data here ? (z, s, t)

Reply:
aPosition is indeed shared with the rest of the shaders, and thus has to have 3 components.

color should still be `[u8; 4]` on the Rust side and declared as `UNORM` kind: https://github.com/gfx-rs/gfx/blob/29d30a1254240a1449fc54a7d146b0928642509d/src/hal/src/format.rs#L417

`aColorTexCoord` should be fixed upstream - https://github.com/servo/webrender/pull/2726